src/Utils/utils.ts

```diff
@@ -31,11 +31,16 @@ export const formatTimeShort = (time: Time) => {
   return format(new Date(`1970-01-01T${time}`), "h:mm a").replace(":00", "");
 };
 
-export const relativeDate = (date: DateLike, withoutSuffix = false) => {
+export const relativeDate = (date: DateLike, withOutSuffix = false): string => {
```

Comment:
<img width="712" alt="image" src="https://github.com/user-attachments/assets/6d962ff0-b5c7-40dd-8859-f55b2a18b488" />


```suggestion
export const relativeDate = (date: DateLike, withutSuffix = false) => {
```